cloudbuild.yaml

```diff
@@ -23,6 +23,9 @@ options:
 # TODO(pavelkalinnikov): Consider pushing this image only on commits to master.
 images: ['gcr.io/$PROJECT_ID/trillian_testbase:latest']
 
+# Cloud Build logs sent to GCS bucket
+logsBucket: 'gs://trillian-cloudbuild-logs'
```

Comment:
The bucket name in GCS has underscores instead of dashes.

Reply:
Created 'trillian-cloudbuild-logs'